planner/core/find_best_task.go

```diff
@@ -519,8 +524,6 @@ func (ds *DataSource) convertToIndexScan(prop *property.PhysicalProperty, candid
 		expectedCnt := math.MaxFloat64
```

Comment:
Remove this line and the if branch as well?

Reply:
em... how to remove this and keep the logic remain same?

Reply:
The `expectedCnt` should always be set to `prop.ExpectedCnt` now, i.e, the `prop.IsEmpty()` would always return true when coming to here, so we can just use `prop.ExpectedCnt` as parameter of `addPushedDownSelection`?

Reply:
I got you, Thanks~